proxy/m2_actions.c

```diff
@@ -1192,6 +1192,16 @@ action_m2_container_propdel (struct req_args_s *args, struct json_object *jargs)
 	return action_sqlx_propdel(args, jargs);
 }
 
+static char *
+_get_property(char **properties, char *key)
```

Comment:
This function has a possible general interest, please move it in `core/str.c` and export it with a name consistent with the remaining of the file (maybe `KV_get_key`).